### PR TITLE
Init model

### PR DESCRIPTION
**Pull request by Bui-Cong-Tri**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/66396)

## WHAT this PR does?
Initialize Model

## Notes Impacted Areas:
- N/A.

## Attachments:
- DB image: 
image

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -12,33 +11,65 @@ class User extends Authenticatable
...
-     */
     protected $fillable = [
-        'name',
+        'username',
         'email',
-        'password',
+        'first_name',
+        'last_name',
+        'is_active',
+        'cart_ids'
```
Comment (ngocnm-0258) [inline app/Models/User.php]: 'cart_ids',

File: app/Models/ProductReview.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+class ProductReview extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'user_id',
+        'product_id',
+        'content',
+        'vote'
```
Comment (ngocnm-0258) [inline app/Models/ProductReview.php]: 'vote',

File: app/Models/Order.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Order extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'user_id',
+        'contact_id'
```
Comment (ngocnm-0258) [inline app/Models/Order.php]: 'contact_id',

**Comment by Bui-Cong-Tri** [comment]
Ready

**Review by ngocnm-0258 — APPROVED** [review]

**Review by ngocnm-0258 — APPROVED** [review]

**Pull request merged**